zebra/zebra_evpn_mh.c

```diff
@@ -2618,14 +2674,23 @@ DEFPY(zebra_evpn_es_id,
 			return CMD_WARNING;
 		}
 
-		if  (!es_lid) {
-			vty_out(vty, "%%Specify local ES ID\n");
-			return CMD_WARNING;
+		if (esi_str) {
+			if (!str_to_esi(esi_str, &esi)) {
```

Comment:
I thought about that but just in case we need to play some tricks for interop by configuring a non-zero type I wanted to leave it more of a guideline. To FRR the es-id is an opaque 10-byte for all practical purposes i.e. there is no type based handling on the rxing side. Iutside of FRR it could mean other things.

Reply:
OK - that's fine